keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -841,8 +843,15 @@ in back-to-back releases.
 -->
 
 #### Alpha
+
 - Feature implemented behind a feature flag
-- The strategy `MinimumCompatibilityVersionStrategy` is implemented
+- The strategy `OldestEmulationVersion` is implemented
+
+#### Beta
+
+- e2e tests
+- Lease pings are parallelized
+- Tests are included for third party strategies
```

Comment:
is the section below correct?

> remove the flag or enable the alpha feature

Reads like "X or not X".

Reply:
Fixed. The mechanism should be completely based on the feature gate and API without other flags.